2020/05-07.md

```diff
@@ -0,0 +1,88 @@
+# 5/7
+
+## [Attendees](https://babeljs.io/team):
+
+- Nicolò ([@nicolo-ribaudo](https://github.com/nicolo-ribaudo))
+- Jùnliàng ([@JLHwung](https://github.com/JLHwung))
+- Brian ([@existentialism](https://github.com/existentialism))
+- Kai ([@kaicataldo](https://github.com/kaicataldo))
+- Justin ([@jridgewell](https://github.com/jridgewell/))
+- Henry ([@hzoo](https://github.com/hzoo))
+
+
+## `targets` to top level
+
+> Feedback on https://github.com/babel/rfcs/pull/2 (Nicolò)
+
+```
+// TL;DR
+{
+  "targets": ">1%, not ie 11",
+  "presets": ["@babel/preset-env"]
+}
+```
+
+- PLEASE REVIEW!
+- In chat: some agreement on keeping the `targets` option as an abstraction (from `preset-env`) vs. only allowing browserslist
+- `targets` should be passed on (resolved to browsers) to plugins/presets so they can be used to inform output (maybe this ties into "assumptions/capabilities" -> example: a transform plugin handles async so output it)
+
+## Optional Chaining AST 
```

Comment:
@JLHwung Were we talking about changing this in Babel, or discussing for ESTree? We should write it in the notes.

Reply:
I think whether we will change `OptionalMemberExpression` depends on which proposal we will back. Since we need more research efforts on building consensus, I don't expect it will change in the near future. But once decision is made, we should state it clearly on the AST changes (renaming/new shape/new properties, etc.)